### DISCUSSION TITLE
Passing standard tracing headers from the initial request

### DISCUSSION THREAD

**Discussion by mityayak** [open-telemetry/opentelemetry-java-instrumentation#4127] [Q&A] [answered]
Is it possible to automatically pass standard tracing headers from the initial request into subsequent calls? Let's say we have initial request in WLS and during request processing some service is using apache or jersey client to make an outbound call. I'm wondering if auto-instrumentation can pass traceparent/tracestate, B3 or some other standard tracing headers into such outbound calls from initial request?

**Comment by jkwatson** [comment, marked as answer]
Yes, definitely. This is expected and intended behavior.

**Reply by mityayak** [reply]
Is it configured somewhere? Like list of headers which are propagated if present on initial request

**Reply by jkwatson** [reply]
https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#propagator is the documentation on propagation options.

**Reply by mityayak** [reply]
Thanks, that's what I was looking for! Can we add a list custom headers via configuration? let's say it's not part of existing propagators headers

**Reply by jkwatson** [reply]
No, that is not a thing that is supported. If you want to propagate addtional headers, you would need to write a custom propagator for that.
